backend/algorithms/validate_term_planner.py

```diff
@@ -30,8 +28,7 @@ def validate_terms(data: ValidPlannerData):
                     "is_accurate": is_answer_accurate,
                     "handbook_note": CACHED_HANDBOOK_NOTE.get(course, ""),
                     "unlocked": unlocked,
-                    "warnings": warnings,
-                    "suppressed": yearIndex + 1 < currYear or (yearIndex + 1 == currYear and termIndex <= pastTerm)
```

Comment:
can you please remove the enumerates as yearIndex and termIndex are now unused 🥺

Reply:
Big if true. Thanks for catching that.